### PR TITLE
reconcile evm 1.5 instruction numbers with working draft and client

### PR DESCRIPTION
**Pull request by gcolvin**

### CODE REVIEW

File: libevmasm/Instruction.h [hunk 1]
```diff
@@ -181,6 +174,15 @@ enum class Instruction: uint8_t
 	LOG2,				///< Makes a log entry; 2 topics.
 	LOG3,				///< Makes a log entry; 3 topics.
 	LOG4,				///< Makes a log entry; 4 topics.
+    
+    JUMPTO = 0xb0,      ///< alter the program counter to a jumpdest -- not part of Instructions.cpp
```
Comment (chriseth) [inline libevmasm/Instruction.h]: Please correct indentation.

**Comment by axic** [comment]
This doesn't seem to match EIP615:
```
We tentatively suggest the following opcodes:

0xB0 JUMPTO
0xB1 JUMPIF
0XB2 JUMPSUB
0xB4 JUMPSUBV
0xB5 BEGINSUB
0xB6 BEGINDATA
0xB8 RETURNSUB
0xB9 PUTLOCAL
0xBA GETLOCAL
```

**Comment by axic** [comment]
Also it seems you forked off a branch in July, hence the tests are confused (circle).

**Comment by axic** [comment]
Okay, rebased, squashed the commits and updated to match cpp-ethereum.

**Review by axic — APPROVED** [review]

**Comment by gcolvin** [comment]
Thanks @axic.  I take it the older branch is not an issue after all?

**Comment by axic** [comment]
It was (it had broken tests), but I've rebased it to head.

**Comment by gcolvin** [comment]
OK, thanks.  I didn't think that old branch had anything to do with this PR - I created a new branch, changed one file, checked that the PR could merge and thought all was well.  (Except, it turns out, for the invisible spaces.)  Oh well.

**Pull request merged**